Frontend/src/components/searchBar/searchBar.module.scss

```diff
@@ -27,7 +27,13 @@
   border-right: 0;
   border-radius: 30px 0 0 30px;
   box-sizing: border-box;
-  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1), inset 0px 2px 3px rgba(0, 0, 0, 0.05);
+  box-shadow: inset 0px 2px 4px rgba(0, 0, 0, 0.1);
+  &:focus {
+    box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1), inset 0px 2px 3px rgba(0, 0, 0, 0.05);
+    +.searchBtn {
```

Comment:
super, pridam ve svem commitu k tomu jeste mezeru mezi + a . tak jen abys o tom vedel, ze to tam patri.